servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -318,16 +321,15 @@ def cromwell_query_params(query, page, page_size):
     return query_params
 
 
-def format_job(job, now):
+def format_job(job, now=datetime.utcnow()):
```

Comment:
I would recommend reverting this, I'm not sure when this actually gets evaluated but I know there are definitely some gotchas with non-primitive default parameters.

Reply:
ok, done

Reply:
Here's more detail on this particular issue: http://blog.thedigitalcatonline.com/blog/2015/02/11/default-arguments-in-python/#default-arguments-evaluation